ios/FluentUI/Navigation/Views/AvatarTitleView.swift

```diff
@@ -5,21 +5,19 @@
 
 import UIKit
 
-// MARK: LargeTitleView
```

Comment:
Ohhh I see now. Given the scope of this change, it might be good to have broken this into two separate PRs. Easier to review, easier to roll back if something goes sideways.